trurl.c

```diff
@@ -420,7 +429,7 @@ static void get(struct option *op, CURLU *uh)
         size_t vlen;
         int i;
         bool urldecode = true;
-        end = strchr(ptr, '}');
+        end = strchr(ptr, endbyte);
         ptr++; /* pass the { */
         if(!end) {
           /* syntax error */
```

Comment:
```suggestion
        if(!end) {
          /* syntax error */
          fputc(startbyte, stream);
```
To make `trurl -g '{scheme} {host' https://localhost` output `https {host` instead of `https host` as mentioned in #132